packages/next-app/src/app/stableswapsimulator/(components)/GraphView.tsx

```diff
@@ -0,0 +1,18 @@
+"use client";
+
+import { useRouter } from "next/navigation";
+
+import { useStableSwap } from "#/contexts/StableSwapContext";
+
+import StableCurve from "./StableCurve";
+
+export function GraphView() {
+  const { initialData } = useStableSwap();
+  const { push } = useRouter();
+
+  if (!initialData.swapFee) {
+    push("/stableswapsimulator");
+  }
```

Comment:
I think I `#would` move this `push` to within the React Context since it's not really a composable component, just somewhere we've decided to concentrate state logic that's related to stableswap.